Source/carthage/Build.swift

```diff
@@ -224,24 +224,44 @@ public enum BuildPlatform {
 	/// Build only for watchOS.
 	case watchOS
 
-	/// The `Platform` corresponding to this setting.
-	public var platform: Platform? {
+	case Multiple([BuildPlatform])
```

Comment:
It seems that `ArgumentType` cannot handle array type, so we need a wrapper type something like:

```swift
public struct BuildPlatformList: ArgumentType {
	let buildPlatforms: [BuildPlatform]

	public static let name = "platform"

	public static func fromString(string: String) -> BuildPlatformList? {
		let commaSeparated = split(string, allowEmptySlices: false) { $0 == "," }

		let findBuildPlatform: String -> BuildPlatform? = { string in
			for (key, platform) in BuildPlatform.acceptedStrings {
				if string.caseInsensitiveCompare(key) == NSComparisonResult.OrderedSame {
					return platform
				}
			}
			return nil
		}

		switch commaSeparated.count {
		case 0:
			return nil

		case 1:
			return findBuildPlatform(commaSeparated[0]).map { BuildPlatformList(buildPlatforms: [ $0 ]) }

		default:
			var buildPlatforms = [BuildPlatform]()
			for string in commaSeparated {
				if let found = findBuildPlatform(string) where found != .All {
					buildPlatforms.append(found)
				} else {
					// Reject if an invalid value is included in the comma-
					// separated string.
					return nil
				}
			}
			return BuildPlatformList(buildPlatforms: buildPlatforms)
		}
	}
}
```

Is this the right direction?